### PR TITLE
Command export emails, code refactor.

### PR DESCRIPTION
**Pull request by nicolasmendoza**
Work in progress.

Related with #1065 . Using export_emails command.

### CODE REVIEW

File: django_extensions/management/commands/export_emails.py [hunk 1]
```diff
@@ -49,72 +58,50 @@ def handle(self, *args, **options):
...
         """CSV format suitable for importing into google GMail
         """
-        csvf = writer(out)
+        csvf = writer(sys.stdout)
         csvf.writerow(['Name', 'Email'])
         for ent in qs:
-            csvf.writerow([full_name(**ent).encode(self.encoding),
-                           ent['email'].encode(self.encoding)])
-
-    def outlook(self, qs, out):
```
Comment (trbs) [inline django_extensions/management/commands/export_emails.py]: Could we keep the `outlook` output ?

**Comment by trbs** [comment]
Thanks looks good. Added a comment on the `outlook` output disapearing

**Comment by trbs** [comment]
Is the PR still WIP ? or could we start merging it ?

File: django_extensions/management/commands/export_emails.py (hunk 1 above)
Comment (nicolasmendoza) [inline django_extensions/management/commands/export_emails.py]: Yes, off course.

**Comment by nicolasmendoza** [comment]
Not ready for merge. I'll upload a final PR soon.

**Comment by nicolasmendoza** [comment]
@trbs. PR is Done.

**Comment by nicolasmendoza** [comment]
Improvements and bugfixes:

- [ ] Some code refactor.
- [ ] "export_emails  -f emails". Now works from PY2.x to PY3.x.
- [ ] "export_emails  -f google". Now works from PY2.x to PY3.x.
- [ ] "export_emails"  -f linkedin. Now works from PY2.x to PY3.x.
- [ ] "export_emails"  -f address. Now works from PY2.x to PY3.x.
- [ ] "export_emails"  -f outlook. Now works from PY2.x to PY3.x.
- [ ] "export_emails"  -f vcard. Fix. Now working with PY3.x, pending fix for PY2.x.
- [ ] Added tests cases for export_emails module. good coverage.

**Comment by trbs** [comment]
Awesome work thanks !

**Pull request merged**